### ISSUE TITLE
The Sigilless variables section is not searchable using a symbol `\`

### ISSUE THREAD

**Issue by titsuki** [perl6/doc#742]
Honestly to say, I'm not sure `\` is addressed as a sigil.

See the following example.
```
$ perl6 -e 'sub foo(\arr){ say arr; }; foo([1,2,3]);'
[1 2 3]
```
The `\` in the variable `\arr` is a sigil in my understanding.

**Comment by AlexDaniel** [comment]
It is documented here: https://docs.perl6.org/type/Scalar. There's a paragraph that starts with “SSA-style constants bind directly to their value”.

I'd say that we should just make it searchable and that's it.

**Comment by titsuki** [comment]
@AlexDaniel Thanks for letting me know that page ! 
BTW then what is the name of `\` ? I'll change the title of this ISSUE.

**Comment by AlexDaniel** [comment]
I have no idea. Is it [“term binding”](http://design.perl6.org/S06.html#Term_binding)?

**Comment by parabolize** [comment]
Sigilless variables. You only need the \ in assignment so its not really a sigil.
https://docs.perl6.org/language/variables#Sigilless_variables
https://docs.perl6.org/language/containers#Binding

**Comment by titsuki** [comment]
@AlexDaniel @parabolize Thanks ! I've changed the title.

**Comment by gfldex** [comment]
fixed with 066b3bab08

There may be more spots where we should index \\, sadly any form of full text search is hopeless in this case.
